integration/mocha-tests/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "sdk-integration-tests",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "clean": "rm -rf temp && rm -rf ../kite-allure-reports && rm -rf logs",
```

Comment:
kite allure? Why is this needed

Reply:
Guess this is copied from Kunal's PR :D. Will removed